website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -42,7 +42,7 @@ Link to GH discussion forthcoming
 
 ### Advanced configurations for incremental models
 
-- The most popular adapters now support an `incremental_predicates` config, to enable greater flexibility when tuning performance in `merge` and `delete` statements for large datasets. ([dbt-labs/docs.getdbt.com/issues/2636](https://github.com/dbt-labs/docs.getdbt.com/issues/2636))
+- [**`incremental_predicates`** config](incremental-models#about-incremental_predicates) is now supported on the most popular adapters, enabling greater flexibility when tuning performance in `merge` and `delete` statements against large datasets.
```

Comment:
@jtcohen6 i've updated the link to the above because the netfliy built URL kept taking me to 'Filtering rows on an incremental run' instead of 'incremental predicates'

Reply:
@mirnawong1 I'm seeing the same behavior. Unfortunately, I think this is an underlying "engineering" issue. If I had to guess, it has to do with the sequence in which:
1. the vertical position of each anchor is determined, and the anchored link goes to that spot
2. the page rendering completes, vertical positions change, the page "moves" from its anchored point

Reply:
thank you!